programmers/난이도별/level02.주식가격/rockmiin.py

```diff
@@ -0,0 +1,14 @@
+def solution(prices):
+    answer=[]
+    for i in range(len(prices)):
+        cnt=0
+        for j in range(i+1, len(prices)):
+            cnt+=1
+            if prices[j]<prices[i]:
+                break
+        answer.append(cnt)
+    return answer
```

Comment:
```suggestion
    pricesLen = len(prices)
    for i in range(pricesLen):
        cnt=0
        for j in range(i+1, pricesLen):
            cnt+=1
            if prices[j]<prices[i]:
                break
        answer.append(cnt)
    return answer
```
len 같이 배열의 길이를 계산하는 것들은 외부에서 선언해주는게 좋습니다!